recipes/fwdpy11/meta.yaml

```diff
@@ -24,13 +24,17 @@ requirements:
     - tskit >=0.1.5
     - gsl
     - openblas
+    - attrs
```

Comment:
@molpopgen, I just assumed the `attr` requirement was supposed to be `attrs`. Is that correct?

Reply:
The conda package is called attr or attrs.  The former is from forge, I believe.

Reply:
Sorry for not giving more context. So, conda-forge has both `attr` and `attrs`. `attrs` is the Python project (https://www.attrs.org/en/stable/) and `attr` is the GNU project http://savannah.nongnu.org/projects/attr.
This being a Python package, I assumed you'd want `attrs` and not the Linux-specific `attr`.

Reply:
https://anaconda.org/anaconda/attrs

https://anaconda.org/conda-forge/attr

However, it looks like the attr version doesn't support macos.  Ugh. On PyPi, both attr and attrs are valid.  That probably explains the macos build failure.

Reply:
> Sorry for not giving more context. So, conda-forge has both `attr` and `attrs`. `attrs` is the Python project (https://www.attrs.org/en/stable/) and `attr` is the GNU project http://savannah.nongnu.org/projects/attr.
> This being a Python package, I assumed you'd want `attrs` and not the Linux-specific `attr`.

Yeah, I just figured that out.  Quite confusing, as you can pip install either name.

Reply:
Yep, the naming is rather confusing, I agree. On top of that, `pip attr` would probably give you yet *another* package (https://pypi.org/project/attr/) o_O.

Reply:
(Plus the `attr.s` thing of course...)

Reply:
Yes, it does seem like a bit of a joke at times.

Reply:
Well, changing attr to attrs now makes the Linux check fail.  But I can't see the logs to know why.

Reply:
So updating the tskit min version. Makes the checks pass now.

Reply:
Seems like this is okay to merge?

Reply:
Should I change the merge target to master instead of the bump branch?  If so, we can close the autobump.

Reply:
Yep, looks all good AFAICT. Thanks for the updates!

Reply:
Also, the "can't see the logs" thing should be fixed now. I had to add a workaround for newer `conda` versions to `bioconda-utils`. Thanks for making us aware of the issue!

Reply:
Great!  Thanks for fixing that.  It makes things a lot easier for systems where you cannot replicate things locally.

Reply:
> cannot replicate things locally.

Most definitely! I hate those things :).